ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/hosts.ini

```diff
@@ -96,6 +96,7 @@ mlag-configuration
 monitor-layer1
 monitor-connectivity
 monitor-sessions
+monitor-sessions-default
```

Comment:
```suggestion
monitor-session-default
```

Reply:
please run the molecule after this, the generated configs and md file name should also be changed

Reply:
fixed!